__tests__/app.test.js

```diff
@@ -34,4 +34,27 @@ describe('GET Requests', () => {
             });
         });
     });
+    test('/api/articles/:article_id 200: responds with valid article object', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((res) => {
+            expect(typeof res.body.article_id).toBe('number');

```

Comment:
We should check that the id is 1, being a number is not enough. We could also be more specific with the rest of the values 🙂